src/shared/utils/diagnose.ts

```diff
@@ -114,7 +114,7 @@ const diagnoseV2 = ({ kind, document, debug, mode }: { kind: Kind; document: any
   try {
     // eslint-disable-next-line @typescript-eslint/no-unused-expressions
     mode === "strict" ? SignatureStrict.check(document.signature) : Signature.check(document.signature);
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
Can use the genetic error.i think message is available

Reply:
Catch clause variable type annotation must be 'any' or 'unknown' if specified.

https://github.com/Microsoft/TypeScript/issues/8677#issuecomment-220385124

Reply:
ok, I thought Error was allowed, but it's true an error can be a string. Let's use unknown => https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables